src/tbbbind/tbb_bind.cpp

```diff
@@ -381,6 +401,40 @@ class system_topology {
         hwloc_bitmap_free(mask_to_free); // If bitmap is nullptr, no operation is performed.
     }
 
+    int get_last_cpu_location() {
+        /* thread_local variable has automatically static-like initialization and lifetime */
+        thread_local raii_affinity_mask last_cpu_location{};
+        int err = hwloc_get_last_cpu_location(topology, last_cpu_location, HWLOC_CPUBIND_THREAD);
+        __TBB_ASSERT_EX((int)err == 0 && hwloc_bitmap_weight(last_cpu_location) == 1, "hwloc_bitmap weight error");
```

Comment:
`hwloc_bitmap_weight` must be in `__TBB_ASSERT`, otherwise it is ineffective, because of (void)element in `__TBB_ASSERT_EX`
```Suggestion
__TBB_ASSERT_EX((int)err == 0, "hwloc_get_last_cpu_location return error code");
__TBB_ASSERT(hwloc_bitmap_weight(last_cpu_location) == 1, "bitmap weight for thread mask should be 1");
```